google/cloud/pubsub/internal/batch_callback_wrapper.h

```diff
@@ -55,6 +55,8 @@ class BatchCallbackWrapper : public BatchCallback {
     child_->EndMessage(ack_id, event);
   };
```

Comment:
s/;//

And while you're here, fix the comment layout on line 30.

Reply:
Done